androidHackathonApp/build.gradle.kts

```diff
@@ -14,6 +15,7 @@ repositories {
 }
 
 val recyclerVersion = "1.1.0"
+val nav_version = "2.3.1"
```

Comment:
Feel free to ignore - just curious, is the snake case preferred on Android compared to camel case? Would be nice to have them the same way

Reply:
Sure, good point :)

Reply:
I'm not sure which way is preferred for versions, will stick with camel case